src/index.html

```diff
@@ -11,7 +11,117 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form id="myForm" action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <div class="group">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label for="surname" class="label-margin">Surname:
+            <input type="text" id="surname" name="surname" 
+            autocomplete="off"
+            >
+          </label>
+          <label for="name" class="label-margin">Name:
+            <input type="text" id="name" name="name" 
+            autocomplete="off"
+            >
+          </label>
+          <label for="age" class="label-margin">How old are You?
+            <input type="number" id="age" name="age" 
+            min="1" max="100" placeholder="12"
+            >
+          </label>
+          <label for="dateOfBirth" class="label-margin">Full date of birth:
+            <input type="date" id="dateOfBirth" name="dateOfBirth">
+          </label>
+          <label class="no-margin">
+            I accept the terms of the agreement
+            <input type="checkbox" name="acceptTerms" required>
+          </label>
+        </fieldset>
+      </div>
+
+      <div class="group">
+        <fieldset>
+          <legend>Registration</legend>
+          <label class="label-inline label-margin" for="email">Email:
+            <input type="email" id="email" name="email" 
+            placeholder="email@example.com" class="input-inline"
+            >
+          </label>
+          <label for="pwd" class="label-margin">Password:
```

Comment:
```suggestion
          <label>Password:
```
The last labels of each fieldset shouldn't have a bottom margin.
Also, you don't need "for" attribute if input is wrapped into label
![image](https://user-images.githubusercontent.com/96409563/225108498-31a2e1a3-14dc-43fe-8421-727528875fb7.png)